src/rdf/graph.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Represents an RDF graph
+ */
+export default class Graph {
+  constructor() {
+    this.statements = []
+  }
+
+  /**
+   * Return a serialization of the graph, maybe in n-triples?
```

Comment:
We've got JSON-LD in Trellis. Not sure what is best for this context. @jermnelson ?

Reply:
That's fine, but JSON-LD is a little harder to write an exporter for, so I'd defer that.